studio/components/ui/Charts/Charts.constants.ts

```diff
@@ -0,0 +1,19 @@
+// Just extracting the color configuration to a single file so that it's easier to observe
+// Needs better naming to be honest, but just to get things going
+
+// For ChartHandler
+export const CHART_COLORS = {
+  TICK: '#6B7280',
+  AXIS: '#444444',
+  GREEN_1: 'rgba(62, 207, 142, 1.0)', // #3ECF8E
+  GREEN_2: 'rgba(62, 207, 142, 0.2)',
+}
+
+export const USAGE_COLORS = {
+  200: '#3ECF8E',
```

Comment:
@Ziinc @joshenlim 

nice, we should be able to use CSS variables in here instead 🥳

Reply:
sounds good! yup we can do this once the [UI library PR](https://github.com/supabase/supabase/pull/5463) is in
just curious though how will this look like in JS?

is it something like `import { colors_scale } from @supabase/ui`?